web-frontend/modules/builder/components/ApplicationBuilderFormulaInput.vue

```diff
@@ -37,9 +38,17 @@ const props = defineProps({
     required: false,
     default: () => [],
   },
+  applicationContextAdditions: {
```

Comment:
Changes in this file are unrelated but they fix the RecordSelector after the advanced formula PR.